### ISSUE TITLE
Suggestion: Specify `<title>` and `<desc>` for certain SVGs

### ISSUE THREAD

**Issue by nchieng** [zendeskgarden/svg-icons#138]
In Admin Center, we utilise the product icon SVG and wordmark SVG on a table of products.

![image](https://user-images.githubusercontent.com/29188449/72396704-abc7b900-3791-11ea-9474-6d8bc7c7cdcd.png)

We had an a11y report that mentioned that these SVGs were not accessible we should either:
* Specify a `<title>` and `<desc>` tag
* If there is accompanying text that describes the SVG, specify an `aria-labelledby`
* Mark them as decorative with `aria-labelledby`.

Was just thinking if it was a good idea to make the `<title>` and `<desc>` available on a select few SVGs that need them? (eg. product icons and wordmarks)

**Comment by austin94** [comment]
We normally avoid providing `title`, `description`, and `alt` for any visual elements since they are used in a variety of scenarios where our default description would be invalid. Localization is also often an issue, but since these are branded it might not be an issue depending on where this is being used.

A few paths I've seen people take are:
- Apply a `role="img"` and include a standard `alt` tag
- Include an `aria-labelledby` ID to a visually hidden element with the correct text

**Comment by jzempel** [comment]
As @austin94 says, we'd be quite apprehensive to apply English text to Garden components (incl SVGs). When there are other options available – we'd prefer those.

**Comment by nchieng** [comment]
That is true. Hadn't thought about localization!
All good, there are definitely alternatives out there that allows you to overwrite the title and desc tags on an SVG.

Closing issue.
